docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3'
 
 services:
+    reporter:
+        image: liberoadmin/reviewer-reporter:${REPORTER_VERSION}
+        environment:
+            MAILER_AUTH_USER: 
+            MAILER_AUTH_PASS:
+            MAILER_HOST:
+            MAILER_PORT:
+            PGHOST:
+            PGUSER:
+            PGPASSWORD:
+            PGDATABASE:
+            PGPORT:
```

Comment:
any `healthcheck`? To be able to add a wait for healthiness later: https://github.com/elifesciences/elife-xpub-formula/blob/master/salt/elife-xpub/init.sls#L177

Reply:
Let me think on this ...

Reply:
... ok I've done some testing and some thinking....

In order to get a good test then the check can be quite complex - and its testing when the last successful run was made and failure of the run is in all likelihood not remedied by restarting.

I'm thinking of skipping health checks because:
1) The reporting is not critical to operations
2) The missing / faulty emails should be apparent relatively quickly
3) Automated restarting of the container will probably not help - more likely to be other infra at fault.

/cc @hdrury1 @giorgiosironi  @elifesciences/xpub